src/ugeneui/ugeneui.pri

```diff
@@ -18,7 +18,20 @@ LIBS += -L../$$out_dir()
 LIBS += -lU2Core$$D -lU2Designer$$D -lU2Algorithm$$D -lU2Formats$$D -lU2Gui$$D -lU2View$$D -lU2Test$$D -lU2Lang$$D -lU2Private$$D -lbreakpad$$D
 LIBS += $$add_sqlite_lib()
 
-macx: LIBS += -framework Foundation /System/Library/Frameworks/Security.framework/Security
+macx {
+    exists( /System/Library/Frameworks/Security.framework/Security ) {
+        LIBS += -framework Foundation /System/Library/Frameworks/Security.framework/Security
+    } else {
+        LIBS += -framework Foundation
+    }
+}
+if (exclude_list_enabled()) {
+    DEFINES += HI_EXCLUDED
+}
+
+contains(DEFINES, HI_EXCLUDED) {
```

Comment:
We do not use HI_EXCLUDED anymore: the final bundler script needs to remove pre-built test libs

Reply:
done here
the scripts will updated later, as they needs to be checked once more separately